server/utils/requestType.ts

```diff
@@ -0,0 +1,29 @@
+import { ServerResponseInterface } from '../graphql/serverResponse'
+import { updateRequestGroupHelper } from './requestGroup'
+
+import { UserInputError } from 'apollo-server-errors'
+
+const updateRequestTypeHelper = (requestType, dataSources, dateUpdated = Date.now()): Promise<ServerResponseInterface> => {
+  if(!requestType.id) {
+    throw new UserInputError('Missing argument value', { argumentName: 'id' })
+  }
+  const requestGroupId = dataSources.requestTypes.getById(requestType.id).requestGroup
+  requestType.dateUpdated = dateUpdated
+  return dataSources.requestTypes.update(requestType)
+    .then(res => {
+      const requestGroup = dataSources.requestGroups.getById(requestGroupId.toString())
```

Comment:
You can update using just the id and the fields to be updated in Mongo. However, I can see why implementing it like this is easier. Just something to keep in mind in the future or if you want the challenge of making your code work in this way. Unless @meganniu or @Anand-Issac  think we should do this way now.

E.g.: pass only id and fields to be updated to the update helper and not the entire requestGroup